### DISCUSSION TITLE
Is there a way to not stop on periods?

### DISCUSSION THREAD

**Discussion by zheckert** [nlp-compromise/nlp_compromise#1123] [Q&A] [answered]
If I wanted to lemmatize a handful of sentences at once, is there a flag or parameter I can pass along to do this without stopping on periods? I've been poring over the documentation but if there's an obvious solution I'm missing it. Feel free to point me at the right thing! I've been loving the library so far, thank you!

Edit: I'm just using `doc.compute('root');`

**Comment by spencermountain** [comment, marked as answer]
hey Zach, yeah - the parser requires splitting sentences, in order to understand the grammar - but you can just do `doc.terms()` to get a list of all words, in all sentences
let me know if I can help further
cheers
